cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_bono.xml

```diff
@@ -0,0 +1,1856 @@
+<cacti>	
```

Comment:
I was planning, as part of my doc updates, to make http://clearwater.readthedocs.org/en/latest/Cacti/ point into this repository at these templates (and remove the current ones).